charts/community-operator/values.yaml

```diff
@@ -90,6 +90,8 @@ resource:
     enabled: false
 
     # Installs Cert-Manager in this cluster.
+    sampleX509User: true
+    useAgentCert: true
```

Comment:
I just added this modification. But now I realise, actually enabling X509 is going to break this template until the next operator release since X509 is not yet available.

Reply:
This should be okay, the helm charts shouldn't change until the next operator release.

Reply:
Good then, fixed this!